repository_service_tuf/cli/__init__.py

```diff
@@ -6,13 +6,17 @@
 import os
 import pkgutil
 import re
+import sys
 from pathlib import Path
 
 import rich_click as click  # type: ignore
+from rich.console import Console
```

Comment:
I noticed in other files you remove those lines:
```
from rich.console import Console
console = Console()
```

and instead use an instance from `rich` directly like this:
```
from repository_service_tuf.cli import console
```
What is the difference between the two and is there a reason why here is different?

Reply:
No difference, as the `cli.__init__.py` is used by them. I decided to import it from the existing instance instead creating multiple console instances for each command file.